tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -852,72 +880,45 @@ QualType GetOrCreateVectorSpecialization(ASTContext& context, Sema* sema,
               llvm::APInt(context.getIntWidth(context.IntTy), colCount), false),
           context.IntTy)};
 
-  QualType vectorSpecializationType = GetOrCreateTemplateSpecialization(context, *sema, vectorTemplateDecl, ArrayRef<TemplateArgument>(templateArgs));
+  QualType vectorSpecializationType = GetOrCreateTemplateSpecialization(
+      context, *sema, vectorTemplateDecl,
+      ArrayRef<TemplateArgument>(templateArgs));
 
 #ifndef NDEBUG
   // Verify that we can read the field member from the template record.
-  DXASSERT(vectorSpecializationType->getAsCXXRecordDecl(), 
+  DXASSERT(vectorSpecializationType->getAsCXXRecordDecl(),
            "type of non-dependent specialization is not a RecordType");
-  DeclContext::lookup_result lookupResult = vectorSpecializationType->getAsCXXRecordDecl()->
-    lookup(DeclarationName(&context.Idents.get(StringRef("h"))));
-  DXASSERT(!lookupResult.empty(), "otherwise vector handle cannot be looked up");
+  DeclContext::lookup_result lookupResult =
+      vectorSpecializationType->getAsCXXRecordDecl()->lookup(
+          DeclarationName(&context.Idents.get(StringRef("h"))));
+  DXASSERT(!lookupResult.empty(),
+           "otherwise vector handle cannot be looked up");
 #endif
 
   return vectorSpecializationType;
 }
 
-
-// Decls.cpp constants start here - these should be refactored or, better, replaced with clang::Type-based constructs.
+// Decls.cpp constants start here - these should be refactored or, better,
+// replaced with clang::Type-based constructs.
 
 static const LPCSTR kBuiltinIntrinsicTableName = "op";
 
-static const ArTypeObjectKind g_ScalarTT[] =
-{
-  AR_TOBJ_SCALAR,
-  AR_TOBJ_UNKNOWN
-};
+static const ArTypeObjectKind g_ScalarTT[] = {AR_TOBJ_SCALAR, AR_TOBJ_UNKNOWN};
```

Comment:
I feel like this is a loss of readability. If we had comments after each entry, would clang-format let us keep the multiple lines?

Reply:
Yes, clang will keep it if there are trailing comments. I don't want to go through and make a bunch of hand edits here to make people happy about individual formatting changes.

This change is about adopting a policy and enforcing it.

Reply:
You don't want to do it, but would you object if I did?

Reply:
I have no objection to fixing formatting issues. You can do it before I merge the clang-format change or after. Either way is fine.